app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        vaccine = visitor["vaccine"]
+        expiration_date = vaccine.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError()
```

Comment:
According to code style point 3 from the checklist, you should provide a custom error message when raising a custom exception. For example: `raise OutdatedVaccineError('The vaccine is outdated.')`